db/migrate/20210129203500_create_posts.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePosts < ActiveRecord::Migration[6.1]
+  def change
+    create_table :posts do |t|
+      t.text :name
+      t.text :description
+      t.references :user, null: false, foreign_key: true
```

Comment:
Good use of association with foreign keys here. Something that I need to get working on mine!